### DISCUSSION TITLE
Modifying existing webhooks

### DISCUSSION THREAD

**Discussion by jwmoss** [community/community#24347] [Apps, API and Webhooks]
<p>We have a handful of webhook URLs spread out in different organizations with the same name. We need to modify this to point to a new server.</p>
<p>Old: <a href="http://jenkins.old.com" rel="noopener nofollow ugc">http://jenkins.old.com</a><br>
New: <a href="http://jenkins.new.com" rel="noopener nofollow ugc">http://jenkins.new.com</a></p>
<p>I’m not seeing an easy way to use the REST API to query this and modify this. Can anyone give me any advice?</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/jwmoss">@jwmoss</a> :wave:</p>
<p>We do have an endpoint for this purpose:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#update-a-repository-webhook" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#update-a-repository-webhook" target="_blank" rel="noopener">Repositories - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>…which is:</p>
<pre><code class="lang-auto">PATCH /repos/{owner}/{repo}/hooks/{hook_id}
</code></pre>
<p>Perhaps you’ve already found this but I hope this is helpful!</p>

**Comment by jwmoss** [comment]
<p>I did figure this out! Thanks!</p>
